js/privatebin.js

```diff
@@ -1230,18 +1248,7 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             $errorMessage = $('#errormessage');
             $loadingIndicator = $('#loadingindicator');
             $statusMessage = $('#status');
-
-            // display status returned by php code, if any (e.g. paste was properly deleted)
-            var serverStatus = $statusMessage.text();
```

Comment:
Sometimes the server sends status/error messages in the file. If you remove this, they might not be displayed. (I did not extensively test it, I just adapted this behaviour from pre-refactoring)